src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -57,6 +58,29 @@ class SQLiteAdapter extends PdoAdapter implements AdapterInterface
 
     protected $suffix = '.sqlite3';
 
+    /** Indicates whether the database library version is at least the specified version
+     *
+     * @param string $ver The version to check against e.g. '3.28.0'
+     * @return boolean
+     */
+    public function databaseVersionAtLeast($ver)
```

Comment:
This function seems to be repeating logic from `version_compare` https://php.net/version_compare

Reply:
That's debatable: SQLite version numbers don't have the same rules as PHP version numbers. It would perform the same functionality, though, I suppose.

Reply:
I think your implementation is the same as `version_compare`, probably will be more clear to use that func

Reply:
I agree. I'll change it.